lib/app/login/login_controller.dart

```diff
@@ -41,14 +29,10 @@ class LoginController extends Controller {
     _loginPresenter.loginOnError = this._loginOnError;
   }
 
-  void _register() {
-    
-  }
-
   /// Login is successful
   void _loginOnComplete() {
     // TODO: Dismiss any loading events then navigate
-    Navigator.of(_context).pushReplacementNamed('/home');
+    Navigator.of(_context).pushNamed('/home');
```

Comment:
Why not replacement @briceallard ?

Reply:
This is the UI glitch I was telling you about. For some reason, on the create account text beneath this button, the Gesture on tap isn't being recognized until AFTER I click Login, and if I go back a screen afterwards, it works fine. Can't figure it out.

Reply:
New Issue created with for bug

Reply:
Bug fixed. It was an error with the passed context value.